lang/java/mapred/src/main/java/org/apache/avro/mapred/tether/TetherJob.java

```diff
@@ -87,9 +87,9 @@ public static void setExecutable(JobConf job, File executable, List<String> args
   /**
    * Extract from the job configuration file an instance of the TRANSPROTO
    * enumeration to represent the protocol to use for the communication
-   * 
+   *
    * @param job
-   * @return
+   * @return - Get the currently used protocol
```

Comment:
```suggestion
   * @return Get the currently used protocol
```
Very minor: unnecessary dash